src/preset_cli/cli/main.py

```diff
@@ -120,7 +120,11 @@ def preset_cli(  # pylint: disable=too-many-branches, too-many-locals, too-many-
     ctx.ensure_object(dict)
 
     # store manager URL for other commands
-    ctx.obj["MANAGER_URL"] = manager_url = URL(baseurl)
+    ctx.obj["MANAGER_URL"] = manager_api_url = URL(baseurl)
+
+    if ctx.invoked_subcommand == "auth":
+        # The user is trying to auth themselves, so skip anything auth-related
+        return
```

Comment:
With this the credentials will never be stored (line 155) when the user runs `preset-cli auth`.

Reply:
`def auth()` is invoked after this method, so that's where they will be stored. Then, the user would call something else, like `superset`, which would fetch the JWT, etc